webapp/components/NotificationList/NotificationList.spec.js

```diff
@@ -3,8 +3,8 @@ import NotificationList from './NotificationList'
 import Notification from '../Notification/Notification'
 import Vuex from 'vuex'
 import Filters from '~/plugins/vue-filters'
-
 import Styleguide from '@human-connection/styleguide'
+import { notifications } from '~/components/utils/Notifications'
```

Comment:
Hm, I see what you're doing here. Personally I would prefer to have a folder `NotificationList.spec/` where I would place the fake data. Now you're using this in many locations, also from other components.

I don't mind having it that way. Probably I would have put that in one components folder and simply imported it in another components spec.